packages/components/src/styles/components/accordion.scss

```diff
@@ -36,75 +59,104 @@ $hds-accordion-item-border-radius: 6px;
     }
   }
 
-  &.hds-accordion-item--contains-interactive {
+  &.hds-accordion-item--contains-interactive,
+  &.hds-accordion-item--is-static {
     box-shadow: var(--token-surface-base-box-shadow);
   }
 }
 
+.hds-accordion-item--type-flush {
+  box-shadow: 0 1px 0 0 var(--token-color-border-primary);
+
+  &:first-child {
+    box-shadow: 0 1px 0 0 var(--token-color-border-primary), 0 -1px 0 0 var(--token-color-border-primary);
+  }
+
+  // we need this specificity to override the focus-ring mixin
+  .hds-accordion-item__button.hds-accordion-item__button--parent-does-not-contain-interactive::before {
+    border-radius: 0;
```

Comment:
I think the focus ring should be inset because our components behave this way (I'm immediately thinking about our listItems). If we were to inset this, I see a couple pixel gap top/bottom in the itemToggle. Can we reduce that gap? Otherwise I'm ok with this approach 👍

Reply:
**RE**: Update to this comment. I made these changes in the Figma component to reflect Cristiano's suggestion. It can be found in the Figma file [here](https://www.figma.com/design/noyY6dUMDYjmySpHcMjhkN/branch/WrZPhAVk6NVpfb5OUyTsHp/HDS-Components?m=auto&node-id=37555-77311&t=g7ZfY27G1y7hXqtO-1)

Reply:
Thanks for sharing the Figma reference @andgen404! Just to make sure I understand correctly, we're planning to have the focus 'inset' for the flush variant, and 'outset' for the card variant, is that right?